### ISSUE TITLE
feat: add [WCAG compliant Color Palette] to [FluentUI library]

### ISSUE THREAD

**Issue by minimalisticMe** [microsoft/fluentui-blazor#2889]
# 🙋 Feature Request

I love that Fluent UI is WCAG compliant. I would like to be able create a color blind friendly color palette for n elements.

## 🤔 Expected Behavior

I have a fixed set of n `FluentBadges` that I want to fill with a different color, lets say `n = 6` different `FluentBadges`.
I want each of these `FluentBadges` to have a different random color that is easily differentiable by color blind people.
When I re-visit  the site, it is okay for the `FluentBadges` to have a different color than before.

When I have a set of `n = 11` different `FluentBadges`, I want them also to have an easily differentiable color.

## 😯 Current Behavior

Currently I create a color palette myself with a fixed number of entries.

## 💁 Possible Solution

Given a number of `n` elements, Fluent UI offers a function that creates a `List<string>` with Hex Color Codes that is compliant with WCAG 2+ for color blind people.
It would be helpful to give hints to the kind of colors desired like `Dark`, `Bright`, `Pastel`, `Normal`, ...

## 🔦 Context

Creating useful color palettes is a more difficult and complex task than I would have ever imagined.
Providing such a library would help me to create random sets of colors that can be recognized as different colors by color blind people.

## 💻 Examples

```
public static List<string> ColorPalette(int numberOfEntries, ColorOptions colorOptions)
{
   ...
}

public enum ColorOptions 
{
   Bright,
   Dark,
   Normal,
   Pastel,
   ...
}
```

**Comment by vnbaaij** [comment]
I don't think this is functionality that we should deliver out of the box but rather something a developer can add him/her self. 
We supply the building blocks to define the colors but not the rest
